astroplan/core.py

```diff
@@ -296,21 +355,45 @@ def ra(self):
         """
         Right ascension.
         """
-        raise NotImplementedError
+        if isinstance(self, FixedTarget):
+            return self.coord.ra
+        raise NotImplementedError()
 
     @property
     def dec(self):
         """
         Declination.
         """
-        raise NotImplementedError
+        if isinstance(self, FixedTarget):
+            return self.coord.dec
+        raise NotImplementedError()
 
 
 class FixedTarget(Target):
     """
     An object that is "fixed" with respect to the celestial sphere.
     """
-
+    def __init__(self, coord, name=None, **kwargs):
+        '''
+        TODO: Docstring.
+        '''
+        if not isinstance(coord, SkyCoord):
+            raise TypeError('Coordinate must be a SkyCoord.')
```

Comment:
Oops, you don't want to do this.  We *do* want to allow people to pass in the "low-level" frame classes like `ICRS` or `AltAz` or whatever.  You want *some* kind of validation, though.  Perhaps the thing to do is to check for a few methods that really should be there?  Like `tranform_to` and `represent_as`?  This follows the python tradition of "duck-typing": when you see a bird that walks like a duck and swims like a duck and quacks like a duck, you call that bird a duck.  Similar with coordinates.